src/RpcServer/RpcServer.Node.cs

```diff
@@ -90,6 +92,9 @@ protected virtual JToken GetVersion(JArray _params)
                 forkJson["blockheight"] = hf.Value;
                 return forkJson;
             }));
+            // rpc settings
+            json["protocol"]["maxiteratorresultitems "] = settings.MaxIteratorResultItems;
+            json["protocol"]["sessionenabled"] = settings.SessionEnabled;
```

Comment:
@roman-khimov ["protocol"]["rpc"] ?

Reply:
Yeah, an additional section makes sense for this.